src/databricks/labs/ucx/framework/logger.py

```diff
@@ -1,5 +1,11 @@
 import logging
 import sys
+from dataclasses import dataclass
+from datetime import datetime
+
+from databricks.labs.ucx.framework.crawlers import SqlBackend
+
+logger = logging.getLogger(__name__)
```

Comment:
This will lead to incorrect imports of "from logging import logger" instead of initializing it properly in every module. Does not belong to this file

Reply:
Agreed, see above.